app/views/gardens/index.html.haml

```diff
@@ -22,43 +21,38 @@
 
   %table.table.table-striped
     %tr
+      %th 
       - unless @owner
         %th Owner
       %th Garden name
-      %th Description
-      %th Location
       %th Area
-      %th Active?
+      %th Location
       %th Plantings
-      %th
 
     - @gardens.each do |garden|
       %tr
+        
+        %td.status
+          - if garden.active 
+            %span.label.label-success Active
+          - else
+            %span.label.label-danger Inactive
+        
         - unless @owner
           %td= link_to garden.owner.login_name, garden.owner
-        %td= link_to garden.name, garden
-        %td= garden.description
+        
+        %td.name
+          = link_to garden.name, garden
+
+        %td.area
+          = render :partial => 'area' , locals: {:garden => garden} 
+            
         %td
           - if ! garden.location.blank?
```

Comment:
Also, here: `if garden.location.present?` or `unless garden.location.blank?`

(Oh, I just realized you didn't write this line... but would be a nice refactor :) )